### ISSUE TITLE
Make a page dedicated to the community reps

### ISSUE THREAD

**Issue by ckipp01** [scala/scala.epfl.ch#259]
This is somewhat related to #56 but more narrowly-scoped. I'm not sure we actually want the individual members from each company listed on the site as that can change somewhat frequently, and it's also not that relevant to the community. The actual companies may be of more interest. _However_, what is relevant and should be front and center is the community rep. After reading the comment in https://github.com/scalacenter/advisoryboard/issues/114 I realized that if someone goes to the Scala Center website there is indication of a community rep. I sort of believe this should be easily viewable that there _is_ a rep and who it is. In reality this should maybe even be visible on the Scala lang site itself.

Here's what I'd like to propose:

1. A dedicated page this site to explain the purpose of the community rep and who that individual is
2. Instructions on how to reach out to this person in the appropriate capacity
3. This page linked in the appropriate places

@eed3si9n seeing that you're our sole community rep, how would you feel about this? I see you created https://github.com/eed3si9n/scalacenter-proposal a while back? Maybe this would be a good opportunity to list things like that.

I'd also like to hear from @SethTisue, @darjutak, and @julienrf if possible. Any thoughts on this?

**Comment by julienrf** [comment]
I am not sure about adding a dedicated page on the site. It could also be an entry in the FAQ, for instance. Would you also add a link to that dedicated page to the top navigation bar?

By the way, beyond just the community representative, I wonder if the website should also cover the “governance” of the Scala Center in general. Currently, you can guess that there is an advisory board if you go to the “Corporate Membership” section, but I think we could do a better job at explaining the governance of the Center (including the role of the community representative) independently of that page.

**Comment by ckipp01** [comment]
> I am not sure about adding a dedicated page on the site. It could also be an entry in the FAQ, for instance. 

The tricky part with just putting into an FAQ section is that you lose out on the discoverability of it. Whereas if we have a dedicated page about it that various other places can link to, we really help the discoverability. If someone comes to the website and it's only part of the FAQ it's very unlikely they'll stumble on it.

> Would you also add a link to that dedicated page to the top navigation bar?

I don't think we necessarily need it to be part of the top navigation, but in various places we could mention the community rep and link to this page. For example I could see there literally being a section on the home page mentioning the community rep with a link.

**Comment by SethTisue** [comment]
> I think we could do a better job at explaining the governance of the Center (including the role of the community representative) independently of that page

Agree, but also, I'd note that it's not necessary to do an amazing, ambitious, complete job on this — certainly not initially! — if it would mean it would take longer to address the most important issue here, which is to make it obvious to the community that they have a rep. It seems so basic and so important to me.

**Comment by eed3si9n** [comment]
I also want to make sure we don't make the position of "community rep" too overblown. It's not a congress person (Americanism for member of parliament) representing the whole of Scala, but more like a person picking up community voices specifically to give advices to Scala Center, like a volunteer at a library asking to get more books from southern hemisphere.

**Comment by SethTisue** [comment]
@ckipp01 maybe it would be sufficient (and much better than nothing) to add a few sentences to https://scala.epfl.ch/contact.html ?

**Comment by SethTisue** [comment]
we could raid e.g. https://www.scala-lang.org/blog/2019/03/18/announcing-new-community-representative.html for wordings, perhaps

**Comment by SethTisue** [comment]
I thought about this one more time, especially after Eugene's remark:

> I also want to make sure we don't make the position of "community rep" too overblown. It's not a congress person (Americanism for member of parliament) representing the whole of Scala, but more like a person picking up community voices specifically to give advices to Scala Center, like a volunteer at a library asking to get more books from southern hemisphere.

We definitely don't want people thinking that Eugene and Zainab are their contacts for any/all Scala concerns. The Ambassadors program is supposed to provide a function something like that, alongside the information we provide at https://scala.epfl.ch/contact.html and https://www.scala-lang.org/community/

So I think maybe the status quo on this might actually be fine? The community reps' existence is documented at https://github.com/scalacenter/advisoryboard and is reiterated in the minutes each quarter.

One issue that does give me pause is the question of proposals. But again, perhaps it is not desirable to over-publicize it. We primarily want to receive proposals from well-connected people who are up to speed on how Scala governance and the Scala community/ecosystem generally works. The goal isn't to shout from the rooftops that every random concern of any random person should be a Center proposal. Many of the things people might want to happen would requiring engineering resources the Center doesn't have, anyway. So maybe it's actually okay to mostly rely on the grapevine, including the community reps themselves, to generate occasional proposals.

In short, I'm somewhat tempted to just close this. wdyt @ckipp01 — is there some modest form of this we could agree on? Something we could PR and merge quickly and call it done?

**Comment by ckipp01** [comment]
So thinking more about this again:

> We definitely don't want people thinking that Eugene and Zainab are their contacts for any/all Scala concerns.

Of course, however the inverse of this is that no one is aware that we have community reps and it's not really clearly defined what their roles are to the public. The more I think about this maybe my issue is more with the latter. If you miss the meeting notes or missed the announcements about the community reps they simply aren't mentioned anywhere.

> The Ambassadors program is supposed to provide a function something like that,

This actually brings me even more confusion. Isn't the ambassador program just that... to promote the Scala language in a more visible and clear way? I wouldn't expect the ambassadors to have to field community feedback and concern the same way that the community reps do.

While I still think it'd be nice to have a dedicated page explaining their roles, I'm fine with what you mentioned about just adding another sentence on the contact page or in the FAQ. It's not a hill I'd die on.

**Comment by SethTisue** [comment]
if the answer is "I prefer *not* to be contacted by random community members unless someone actually has a draft of a proposal they want submitted", I think we could figure out a way to accommodate that, too

**Comment by eed3si9n** [comment]
I created a discussion thread a while back - https://github.com/eed3si9n/scalacenter-proposal/discussions
People can contact me on Mastodon or Bluesky as well.

**Comment by zainab-ali** [comment]
The discussions thread is a great idea.

People can contact me on Discord or LinkedIn.  I'm reachable on Bluesky and Mastadon too.

**Comment by SethTisue** [comment]
PR: #391
